pkg/node/sendoperation/sendoperation.go

```diff
@@ -24,6 +24,9 @@ const AccountCreationStorageCost = 10_000_000
 
 const StorageCostPerByte = 1_000_000
 
+// To be updated when key creation cost will be charged for actual size.
+const StorageKeyCreationCost = 10 * StorageCostPerByte
```

Comment:
Why 10 ?

Reply:
its the fixed storage cost of key creation in datastore

Reply:
Maybe be you should add a comment or even better replace 10 with an explicite constant name